packages/ckeditor5-image/src/imageresize/imageresizeediting.ts

```diff
@@ -97,6 +97,7 @@ export default class ImageResizeEditing extends Plugin {
 	 */
 	private _registerConverters( imageType: 'imageBlock' | 'imageInline' ) {
 		const editor = this.editor;
+		const imageUtils = editor.plugins.get( 'ImageUtils' );
```

Comment:
```suggestion
		const imageUtils: ImageUtils = editor.plugins.get( 'ImageUtils' );
```

`ImageUtils` should be imported to make type inference work here. I know it is, but as a matter of style, enforce the need of importing it here.
I know it's imported, but let's enforce

Reply:
Done in 32104408ac4e8910715b4685b369d66a6dda8c34.